tox.ini

```
@@ -25,12 +23,9 @@ commands =
    --cov --cov-report xml:reports/coverage-{envname}.xml \
    {posargs}
 
-[testenv:isort]
+[testenv:ruff]
 extras = tests
 skipsdist = True
-commands = isort --check-only --diff objectsapiclient/ testapp/ tests/
-
-[testenv:black]
-extras = tests
-skipsdist = True
-commands = black --check objectsapiclient testapp tests setup.py
+commands =
+    ruff check objectsapiclient/ testapp/ tests/ setup.py
+    ruff format --check objectsapiclient/ testapp/ tests/ setup.py
```

In my view it's generally preferable to follow a "lint everything" approach, with explicit ignores for files that are matched but should be skipped (not also [that for instance `.gitignore` is observed by ruff](https://docs.astral.sh/ruff/configuration/#python-file-discovery)).


```suggestion
    ruff check .
    ruff format .
```